irohad/ametsuchi/impl/postgres_command_executor.hpp

```diff
@@ -16,6 +16,24 @@ namespace soci {
 
 namespace shared_model {
   namespace interface {
+    class AddAssetQuantity;
+    class AddPeer;
+    class AddSignatory;
+    class AppendRole;
+    class CreateAccount;
+    class CreateAsset;
+    class CreateDomain;
+    class CreateRole;
+    class DetachRole;
+    class GrantPermission;
+    class RemovePeer;
+    class RemoveSignatory;
+    class RevokePermission;
+    class SetAccountDetail;
+    class SetQuorum;
+    class SubtractAssetQuantity;
+    class TransferAsset;
+    class CompareAndSetAccountDetail;
```

Comment:
it is almost sorted by latin alphabet - why not fix the order?

Reply:
I can, but it will not be forced by clang-format, and probably might get mixed again.